### PR TITLE
fix(ui): Fix Indicators editor, add possibility of default sorting in TableCustom and add aria-labels

### PR DESCRIPTION
**Pull request by bdauria**

### CODE REVIEW

File: src/InputField/InputFieldSelectTableCell/index.jsx [hunk 1]
```diff
@@ -19,6 +19,25 @@ class InputFieldSelectCustom extends Component {
     selected: {},
   };
 
+
+  componentWillMount = () => {
+    const { value, options } = this.props;
+    if (options) {
+      for (let i = 0; i < options.length; i += 1) {
```
Comment (kduret) [inline src/InputField/InputFieldSelectTableCell/index.jsx]:
```suggestion
      for (const i = 0; i < options.length; i += 1) {
```

File: src/MaterialComponents/Snackbar/Error/withErrorSnackbar.jsx [hunk 2]
```diff
@@ -16,10 +16,29 @@ const withErrorSnackbar = (Component) => {
...
+    const showErrors = (errors) => {
+      const errorKeys = Object.keys(errors);
+
+      const formattedErrors = errorKeys.reduce(
+        (acc, current) => [...acc, `${current}: ${errors[current]}`],
+        [],
+      );
+  
+      showError(
+        <div style={{ display: 'block' }}>
```
Comment (kduret) [inline src/MaterialComponents/Snackbar/Error/withErrorSnackbar.jsx]: could you move style in a separate object and use useStyles ?

File: src/MaterialComponents/Snackbar/Error/withErrorSnackbar.jsx [hunk 3]
```diff
@@ -16,10 +16,29 @@ const withErrorSnackbar = (Component) => {
...
+          {formattedErrors.map((err, index) => (
+            <p style={{ margin: 0 }} key={errorKeys[index]}>
```
Comment (kduret) [inline src/MaterialComponents/Snackbar/Error/withErrorSnackbar.jsx]: could you move style in a separate object and use useStyles ?

File: src/MaterialComponents/Snackbar/Error/withErrorSnackbar.jsx (hunk 2 above)
Comment (bdauria) [reply src/MaterialComponents/Snackbar/Error/withErrorSnackbar.jsx]: Actually the block is displayed statically so there is no way to use Hooks.

**Pull request merged**